plugins/main/public/controllers/management/components/management/cdblists/views/list-editor.tsx

```diff
@@ -45,7 +46,14 @@ import { UI_ERROR_SEVERITIES } from '../../../../../../react-services/error-orch
 import { UI_LOGGER_LEVELS } from '../../../../../../../common/constants';
 import { getErrorOrchestrator } from '../../../../../../react-services/common-services';
 
+type FieldTypes = 'key' | 'value' | 'edit';
+type FieldStateTypes = 'addingKey' | 'addingValue' | 'editingValue';
+
 class WzListEditor extends Component {
+  messages = {
+    quotesError: 'Must start and end with quotes or have no quotes at all',
+  };
```

Comment:
```suggestion
  private readonly MESSAGE_QUOTES_ERROR = 'Must start and end with quotes or have no quotes at all';
```

Reply:
With the last change I have added another message.
I add the `private readonly`

Reply:
Done [82787a9](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7171/commits/82787a96f4f8505d0effd3345f9985cb00a2fd41)